src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
```

Comment:
You should create variables for:

1. cost renting per day(40$)
2. discounts
3. days for both discounts(3, 7)

Now it simple program, but in the future if it will be more complicated and you just want to change for example: days for discounts, you have to change it all places where you used it, but with variable you just can change it in one place, where you initialize. So, you should use variables.